projects/aca-shared/src/lib/components/page-layout/page-layout.component.scss

```diff
@@ -7,10 +7,69 @@
     display: flex;
     align-items: center;
     flex: 0 0 65px;
-    flex-basis: 48px;
-    background: var(--theme-background-color);
+    flex-basis: 96px;
+    background: var(--theme-page-layout-header-background-color);
     border-bottom: 1px solid var(--theme-border-color, rgba(0, 0, 0, 0.07));
     padding: 0 24px;
+
+      .adf-breadcrumb-item {
+        font-size: 20px !important;
+        font-weight: 400 !important;
+        letter-spacing: 0.15px !important;
+      }
+
+      .app-search-input {
```

Comment:
why this is in the page-layout styles and not in the app-search-input component styles?

Reply:
aca-search-input is conditionally rendered depending on the route. So when it is rendered with router '/search', it has a grey background and when it is rendered with routes other than '/search', (along with the text buttons), its background is supposed to be transparent.
So .app-search-input styles are added at both places.